apps/www/_alternatives/supabase-vs-heroku-postgres.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Supabase vs Heroku Postgres
+description: Supabase vs Heroku Postgres
+author: ant_wilson
+image: all-pull-together/all-pull-together-thumb.png
+thumb: all-pull-together/all-pull-together-thumb.png
+tags:
+  - community
+  - case-study
+  - q-and-a
+date: '2022-05-26'
+toc_depth: 3
+---
+
+## What is Heroku Postgres?
+
+Heroku is a cloud application platform that offers managed PostgreSQL as a service. They offer 5 levels of Postgres support from the Hobby Tier up to the Shield Tier, each with different levels of features and pricing.
+
+## What is Supabase?
+
+Supabase also offers managed Postgres, the main difference is that with each deployment you also get:
+
+- Auto-generated API - [never write an API again](https://supabase.com/docs/guides/api#rest-api-2)
+- Realtime - [subscribe to data changes via websockets](https://supabase.com/docs/reference/dart/subscribe)
+- Auth - [users can log in and out of your application](https://supabase.com/auth)
+- Functions - [deploy custom logic to the edge](https://supabase.com/edge-functions)
+- Storage - [serve large files and folders](https://supabase.com/storage)
+- PgBouncer - [connection pooling useful for serverless computing](https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool)
+
+## How are they similar?
+
+Heroku Postgres and Supabase both offer:
+
+- [A web UI for managing your instance](https://supabase.com/docs/guides/database#table-view)
+- SLAs and Enterprise-grade support packages
+- Direct SSL connections to Postgres
+- Postgres Extensions (see [Supabase Extensions](https://supabase.com/docs/guides/database/extensions), [Heroku Extensions](https://devcenter.heroku.com/articles/heroku-postgres-extensions-postgis-full-text-search))
+- [Backups and PITR](https://supabase.com/blog/2020/08/02/continuous-postgresql-backup-walg) (not on Heroku’s Hobby tier)
+- [Postgres logs](https://supabase.com/docs/guides/platform/logs) (not on Heroku’s Hobby tier)
+- Encryption-at-rest (not on Heroku’s Hobby tier)
+
+## What are the differences?
+
+### Core Features
+
+Both solutions run PostgreSQL, but in a time when Developer Experience matters, there is a lot you can do to improve the speed at which developers can build products faster and with less human resource. These are some of the key differences between Heroku Postgres and Supabase in terms of features:
```

Comment:
> there is a lot you can do to improve the speed at which developers can build products faster and with less human resource

Not sure I like this - it's subjective and "implies" that supabase is better without giving facts about why/how